src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Semantic/Services/TagHelperSemanticRangeVisitor.cs

```diff
@@ -286,6 +286,18 @@ public override void VisitMarkupTagHelperAttribute(MarkupTagHelperAttributeSynta
             AddSemanticRange(node.ValueSuffix, RazorSemanticTokensLegend.MarkupAttributeQuote);
         }
 
+        public override void VisitMarkupTagHelperAttributeValue(MarkupTagHelperAttributeValueSyntax node)
+        {
+            if(node.Children.First().Kind == SyntaxKind.MarkupTextLiteral)
```

Comment:
I realized just now that this could be something like `="literal @variable more literal"`, gonna rework this into a loop.